src/odemis/driver/technolution.py

```diff
@@ -708,10 +707,12 @@ def _calc_calibration_sampling_period(self, total_line_scan_time, system_clock_p
 
     def getTicksScanDelay(self):
         """
-        :return: Scan delay in multiple of ticks of the ebeam scanner clock frequency
+        Convert the scan delay in seconds into a multiple of the system clock period in ticks.
+        Convert the number of pre-scanned lines to integer.
+        :return (int, int): Scan delay in ticks. Number of line scans before starting the actual acquisition.
```

Comment:
It's not clear that the first sentence is for the first int, and the second sentence for the second int.
I'd suggest to write on multiple lines, like:
```python
return:
   x_delay (int): Scan delay in ticks.
   y_delay (int): Number of line scans before starting the actual acquisition.
```